Kernel/FileSystem/VirtualFileSystem.cpp

```diff
@@ -822,30 +822,46 @@ ErrorOr<void> VirtualFileSystem::symlink(Credentials const& credentials, StringV
     return {};
 }
 
+// https://pubs.opengroup.org/onlinepubs/9699919799/functions/rmdir.html
 ErrorOr<void> VirtualFileSystem::rmdir(Credentials const& credentials, StringView path, Custody& base)
 {
     RefPtr<Custody> parent_custody;
     auto custody = TRY(resolve_path(credentials, path, base, &parent_custody));
     auto& inode = custody->inode();
 
-    // FIXME: We should return EINVAL if the last component of the path is "."
-    // FIXME: We should return ENOTEMPTY if the last component of the path is ".."
+    auto last_component = KLexicalPath::basename(path);
```

Comment:
Nitpick: The final return statement of the function can now be changed to

`return parent_inode.remove_child(last_componet);`

The compiler can hopefully optimize out multiple calls to `KLexicalPath::basename(path);` by itself, but I often find it easier to read code without duplicated logic.